uxarray/grid.py

```diff
@@ -160,33 +160,69 @@ def __from_ds__(self, dataset):
             raise RuntimeError("unknown file format: " + self.mesh_filetype)
         dataset.close()
 
-    def write(self, outfile, grid_type):
+    def outfile_encoder(self, outfile, file_type):
+        """Writes xarray.Dataset to new file in current directory in specified
+        file type.
+
+        Parameters
+        ----------
+        outfile : str, required
+
+        file_type : str, required
+            New file type to be created from write function. Supported options are
+            "netcdf" and "zarr"
+        """
+        if file_type == "netcdf":
+            self.ds.to_netcdf(outfile)
```

Comment:
Intention is not to write the `self.ds` to the output files. Our aim is to write out the dataset that was returned out of the encoders that got in `self.ds`.